components/brave_sync/brave_profile_sync_service_impl.cc

```diff
@@ -424,7 +424,7 @@ void BraveProfileSyncServiceImpl::OnGetInitData(
 
   client_data::Config config;
   config.api_version = brave_sync_prefs_->GetApiVersion();
-  config.server_url = "https://sync.brave.com";
+  config.server_url = "https://sync-staging.brave.com";
```

Comment:
This will be removed when https://github.com/brave/sync/pull/350 is merged into master.
And also update DEPS when that happens before merging

Reply:
updated deps and pointed it to production